include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -50,21 +58,72 @@ class BrushedMotorInterruptHandler {
         return queue.has_message_isr();
     }
 
+    // upon advice from hardware, 0.01mm is a good limit for percision
+    auto acceptable_position_error() -> int32_t {
+        return int32_t(gear_conf.get_encoder_pulses_per_mm() * 0.01);
```

Comment:
this is the only place we ever use this; can we make `acceptable_position_error` be a value parameter to this class's ctor instead and save a float calculation?